components/lib/crash/src/main/java/mozilla/components/lib/crash/handler/CrashHandlerService.kt

```diff
@@ -29,10 +31,18 @@ class CrashHandlerService : IntentService(WORKER_THREAD_NAME) {
 
         intent.extras?.let { extras ->
             val crash = Crash.NativeCodeCrash.fromBundle(extras)
+            val skipPrompt = extras.getBoolean(Crash.INTENT_EXTRA_SKIP_PROMPT, false)
+
+            if (skipPrompt) {
+                val notificationManager = getSystemService(Context.NOTIFICATION_SERVICE)
+                        as NotificationManager
+                notificationManager.cancelAll()
+                sendBroadcast(Intent(Intent.ACTION_CLOSE_SYSTEM_DIALOGS))
```

Comment:
What's the reason for this broadcast here?  🤔

Reply:
Once the report button is clicked, it retracts the notification drawer.  I will agree if you think this is not needed.